helm/aws-operator/templates/deployment.yaml

```diff
@@ -9,6 +9,10 @@ metadata:
 spec:
   replicas: 1
   revisionHistoryLimit: 3
+  selector:
+    matchLabels:
+      app: {{ .Values.project.name }}
+      version: {{ .Values.project.version }}
```

Comment:
The selector has to be specified when using `apps/v1` and this is what tiller was complaining about. 

I think for VOO operators we need to include the version label.